src/app/pages/component-category-list/component-category-list.html

```diff
@@ -1,5 +1,6 @@
 <div class="docs-component-category-list">
   <md-card
+      role="link"
```

Comment:
In general it's always better to use the appropriate native element instead of a role. An `<a>` also has the proper associated `href` and tab index.